adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -274,9 +275,9 @@ public void testFavorLocalCacheUseLocalRTFailsWithInvalidGrantSwitchToBroker()
      * will switch to broker.
      */
     @SmallTest
-    public void testFavorLocalCacheUseLocalRTSucceeds() throws PackageManager.NameNotFoundException,
-            NoSuchAlgorithmException, OperationCanceledException, IOException, AuthenticatorException,
-            InterruptedException {
+    public void testFavorLocalCacheUseLocalRTSucceeds()
+            throws PackageManager.NameNotFoundException, OperationCanceledException,
```

Comment:
@weijjia It looks like she is removing runtime exceptions from the throws declaration.  Is there a reason you think these should be here or are you concerned that this is not directly related to the feature under discussion.